nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java

```diff
@@ -1063,8 +1063,41 @@ public static boolean isValidNumericFilter(FilterMode mode) {
 		/**
 		 * <p>Apply styles to display value. This controls the display of value
 		 * with images and fonts
-		 * <p>When set to true, CSS classes added into <ul><li>{@link app_style.css}</li></ul> in will be added to the container element surrounding this component.
+		 * <p>When set to true, CSS classes added into the application's defined styles will be added to the container element surrounding this component.
 		 * This can be used to apply additional styling, if necessary.
+		 * 
+		 * <p>
+		 * <strong>Example Configuration</strong>
+		 * </p>
+		 * <pre>
+		 * <code>@GridColumn(applyValueStyles = true)
+		 * private String status;
+		 * </code>
+		 * 
+		 * <p>
+		 * <strong>Resulting HTML</strong>
+		 * </p>
+		 * <pre>
+		 * <pre>
+		 * <code>
+		 * <span class="Active status ng-star-inserted" title="Active">
+		 *	    ...
+		 * </span>
+		 * </pre>
+		 * </code>
+		 * </pre>
+		 * 
+		 * <p>
+		 * <strong> Example CSS </strong>
+		 * </p>
+		 * <pre>
+		 * <code>
+		 * .Active {
```

Comment:
It's probably best to make this as .status.Active to promote somewhat better CSS styles.